packages/carbon-component-mapper/src/common/is-required.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { childrenPropTypes } from '@data-driven-forms/common/src/prop-types-templates';
+
+import './style.scss';
```

Comment:
I would call this file same as this component: `is-required.scss`

Reply:
I was inspired by the ant mapper, where it is done the same way. Should I really be inconsistent?

Reply:
So it's wrong there. :smile: